qiskit_nature/deprecation.py

```diff
@@ -228,6 +228,61 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def deprecate_positional_arguments(
+    version: str,
+    func_name: str,
+    pre_positional_kw: List[str],
+    additional_msg: Optional[str] = None,
+    stack_level: int = 3,
+) -> Callable:
+    """Decorator to convert positional arguments into keyword arguments and warn upon use.
+    .. code-block:: python
+
+        @deprecate_positional_arguments("0.1", (b, d))
```

Comment:
This example appears invalid because `func_name` is not provided

Reply:
Also, the `(b, d)` will likely not work as intended.. See also next comment.

Reply:
I made a change in this function and forgot to change the documentation at some point. Thank you for spotting it.